cli/tests/test_sign_unsign_commands.rs

```diff
@@ -349,28 +349,54 @@ backend = "test"
     [EOF]
     ");
 
+    // Unsign a single commit
+    let output = work_dir.run_jj(["unsign", "-r", "@"]);
+    insta::assert_snapshot!(output, @r"
+    ------- stderr -------
+    Unsigned 1 commits:
+      zsuskuln ea304b8a (empty) (no description set)
+    Working copy  (@) now at: zsuskuln ea304b8a (empty) (no description set)
+    Parent commit (@-)      : kkmpptxz 7b6ad8e6 (empty) three
+    [EOF]
+    ");
+
+    let output = work_dir.run_jj(["log", "-r", "all()"]);
+    insta::assert_snapshot!(output, @r"
+    @  zsuskuln test.user@example.com 2001-02-03 08:05:13 ea304b8a
+    │  (empty) (no description set)
+    ○  kkmpptxz test.user@example.com 2001-02-03 08:05:11 7b6ad8e6 [✓︎]
+    │  (empty) three
+    ○  rlvkpnrz test.user@example.com 2001-02-03 08:05:11 8dc06170 [✓︎]
+    │  (empty) two
+    ○  qpvuntsm test.user@example.com 2001-02-03 08:05:11 fbef1f02 [✓︎]
+    │  (empty) one
+    ◆  zzzzzzzz root() 00000000
+    [EOF]
+    ");
+
+    // Unsign multiple commits, including a previously-unsigned revision
     let output = work_dir.run_jj(["unsign", "-r", "..@"]);
```

Comment:
nit: It's more interesting to see rebased descendants "keep"s signed state. For example, if the `@` revision was signed, `unsign -r ..@-` will rewrite the `@` revision, which will have to be re-signed.